dashboard/app/app_test.go

```diff
@@ -512,8 +512,9 @@ var testConfig = &GlobalConfig{
 			},
 		},
 		"tree-tests": {
-			AccessLevel: AccessPublic,
-			Key:         "treeteststreeteststreeteststreeteststreeteststreetests",
+			AccessLevel:           AccessPublic,
+			FixBisectionAutoClose: true,
```

Comment:
You should probably also test for `FixBisectionAutoClose: false`?
Or is it already covered by some other test?

Reply:
Good point!
I've adjusted an existing test to check we don't close bugs if `FixBisectionAutoClose` is false.